src/ophyd_async/epics/_backend/_p4p.py

```diff
@@ -109,6 +182,11 @@ def write_value(self, value):
 
 @dataclass
 class PvaEnumConverter(PvaConverter):
+    """To prevent issues when a signal is restarted and returns with different enum
+    values or orders, we put treat an Enum signal as a DBR_STRING, and cache the
+    choices on this class.
+    """
+
```

Comment:
DBR_STRING is a CA thing, so this docstring seems wrong. Is it trying to convey something important?

Reply:
@DiamondJoseph

Reply:
DBR_STRING -> string?

Reply:
Done.